example/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [
+    ["env", {
```

Comment:
I don't think we need this preset and `transform-decorators-legacy` since we already transpile the code by TypeScript. Can we remove them from babelrc?